ReactiveXComponent/Connection/IXCSession.cs

```diff
@@ -6,7 +6,9 @@ namespace ReactiveXComponent.Connection
     public interface IXCSession : IDisposable
     {
         bool IsOpen { get; }
+        event EventHandler SessionOpened;
```

Comment:
Why not Action instead of EventHandler ? It seems that you don't use the sender and EventArgs are empty most of the time

Reply:
I'll refactor this in the next version as I try to avoid any impact on the modules using this Api (like Client Api generated by the XCStudio).